src/services/passwordGeneration.service.ts

```diff
@@ -25,6 +25,7 @@ const DefaultOptions = {
     type: 'password',
     numWords: 3,
     wordSeparator: '-',
+    commonRequirements: false,
```

Comment:
I would prefer these two be independent options that a user could set from the UI. `capitalize` and `includeNumber` or something.